test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -833,6 +833,64 @@ namespace StackTest {
             $LASTEXITCODE | Should -Be $ExitCodeBadCommandLineParameter
         }
     }
+
+    Context "Startup banner text tests" -Tag Slow {
+        BeforeAll {
+            $outputPath = "Temp:\StartupBannerTest-Output-${Pid}.txt"
+            $inputPath  = "Temp:\StartupBannerTest-Input.txt"
+            "exit" > $inputPath
+
+            # Not testing update notification banner text here
+            $oldPowerShellUpdateCheck = $env:POWERSHELL_UPDATECHECK
+            $env:POWERSHELL_UPDATECHECK = "Off"
+
+            # Set TERM to "dumb" to avoid DECCKM codes in the output
+            $oldTERM = $env:TERM
+            $env:TERM = "dumb"
```

Comment:
Should this be done only on non-Windows?

Reply:
It probably only **needs** to be done non-WIndows since the DECCKM failures I saw where on mac/linux.  That said, I don't believe it hurts to set TERM="dumb" on Windows.